spec/kickstart/main.fmf

```diff
@@ -0,0 +1,71 @@
+story:
+    As a tester I want to specify detailed installation of a guest
+    using the kickstart script.
+
+description: |
+    As part of the :ref:`/spec/plans/provision` step it is
+    possible to use the ``kickstart`` key to specify additional
+    requirements for the installation of a guest. It is possible
+    to specify a kickstart script that will for example specify
+    specific partitioning.
+
+    Introduction
+    ^^^^^^^^^^^^
+
+    The structure of a kickstart file can be separated into several
+    sections.
+
+    The ``pre-install`` key corresponds to the `%pre` section of a file.
+    It can contain BASH commands, this part is run before
+    the installation of a guest.
+
+    The ``post-install`` key corresponds to the `%post` section of a file.
+    It can contain BASH commands, this part is after
+    the installation of a guest.
+
+    The ``script`` key contains the kickstart specific commands that
+    are run during the installation of a guest.
+
+    It is also possible to specify ``bkr-metadata`` that can be read by
+    beaker.
+
+
+    Notes
+    -----
+
+    The implementation for the ``kickstart`` key is in
+    progress. Support of a kickstart file is currently
+    limited to Beaker provisioning pool, that is
+    being  handled by Artemis.
```

Comment:
I'd probably change the wording here, to make it clear it's Beaker in any case, not just with Artemis. For example:

> Support of a kickstart file is currently limited to Beaker provisioning, as implemented by tmt's `beaker` and `artemis` plugins, and may not be fully supported by other provisioning plugins in the future.

Reply:
Changed